python/plugins/processing/algs/qgis/EquidistanceBuffer.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+
+"""
+***************************************************************************
+    EquidistanceBuffer.py
+    --------------------
+    Date                 : April 2019
+    Copyright            : (C) 2019 by Ujaval Gandhi
+    Email                : ujaval at spatialthoughts dot com
+***************************************************************************
+*                                                                         *
+*   This program is free software; you can redistribute it and/or modify  *
+*   it under the terms of the GNU General Public License as published by  *
+*   the Free Software Foundation; either version 2 of the License, or     *
+*   (at your option) any later version.                                   *
+*                                                                         *
+***************************************************************************
+"""
+
+__author__ = 'Ujaval Gandhi'
+__date__ = 'April 2019'
+__copyright__ = '(C) 2019, Ujaval Gandhi'
+
+# This will get replaced with a git SHA1 when you do a git archive323
+
+__revision__ = '$Format:%H$'
+
+from qgis.core import (QgsGeometry,
+                       QgsPoint,
+                       QgsFeature,
+                       QgsWkbTypes,
+                       QgsProcessing,
+                       QgsProcessingParameterDistance,
+                       QgsProcessingParameterNumber,
+                       QgsProcessingParameterEnum,
+                       QgsProcessingException,
+                       QgsCoordinateReferenceSystem,
+                       QgsCoordinateTransform,
+                       QgsProject,
+                       QgsUnitTypes
+                       )
+
+from processing.algs.qgis.QgisAlgorithm import QgisFeatureBasedAlgorithm
+
+
+class EquidistanceBuffer(QgisFeatureBasedAlgorithm):
+    """
+    This algorithm creates equidistance buffers for vector layers.
+
+    Each geometry is transformed to a Azimuthal Equidistant projection centered
```

Comment:
Is it **a** Azimuthal or **an** Azimuthal?

Reply:
I wrote it thinking 'a' refers to the projection and not azimuthal. A "Azimuthal equidistant' projection. I am not sure which is right though.

Reply:
> I am not sure which is right though.

me neither ;)

Reply:
An, if a vowel is after 'a ' it becomes 'an '.

Reply:
Thanks. @roya0045 and @DelazJ . I have replaced 'a' with 'an'.